### PR TITLE
Confetti polish

### PR DESCRIPTION
**Pull request by Duncid**
## Description
- Fine tuning confetti on Onboarding
- Adding "Confetti" to Sparkle
- Bumping the version
- Make Dialog compatible

### CODE REVIEW

File: sparkle/src/components/Dialog.tsx [hunk 1]
```diff
@@ -85,6 +92,14 @@ export function Dialog({
                     {isSaving && <Spinner />}
                   </Button.List>
                 </div>
+                {hasBackground != "none" && (
```
Comment (flvndvd) [inline sparkle/src/components/Dialog.tsx:95]:
```suggestion
                {hasBackground !== "none" && (
```

File: sparkle/src/components/Dialog.tsx [hunk 2]
```diff
@@ -28,7 +29,10 @@ export function Dialog({
   validateVariant = "primary",
   title,
   isSaving,
+  hasBackground = "none",
```
Comment (flvndvd) [inline sparkle/src/components/Dialog.tsx:32]: `hasBackground` sounds more like a boolean value, can we update to `backgroundType` instead?

File: sparkle/src/components/ConfettiBackground.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    }
+
+    return () => {
+      if (resizeObserver.current && referentSize && referentSize.current) {
+        resizeObserver.current.unobserve(referentSize.current);
+      }
+    };
+  }, [referentSize]);
+
+  const confettiProps = {
```
Comment (flvndvd) [inline sparkle/src/components/ConfettiBackground.tsx:66]: We should not define this in the component, as will be recreated every time, let's move it above the component and only add the width and height in the switch/case below.

File: front/pages/w/[wId]/welcome.tsx [hunk 4]
```diff
@@ -247,7 +247,19 @@ export default function Welcome({
         headerTitle="Joining Dust"
         headerRightActions={<></>}
       >
-        <Confetti wind={0.02} gravity={0.08} />
+        <Confetti
```
Comment (flvndvd) [inline front/pages/w/[wId]/welcome.tsx:250]: Can we use the `ConfettiBackground` here?

Comment (Duncid) [reply front/pages/w/[wId]/welcome.tsx:250]: That's the idea.
But it needs to be deployed first.

**Comment by Duncid** [comment]
@flvndvd think we're good.
Can I let you take it from here?
My dev environment does not compile these days, can't test front.

**Comment by flvndvd** [comment]
Yep will take care of it, also happy to seat down with you tomorrow a few minutes to solve you dev environment.

**Comment by Duncid** [comment]
> Yep will take care of it, also happy to seat down with you tomorrow a few minutes to solve you dev environment.

Thanks, will do

**Pull request merged**